web/src/components/TestSpecForm/useExpectedInputLanguage.tsx

```diff
@@ -25,7 +25,9 @@ export function useExpectedInputLanguage() {
             const identifierText = state.doc.sliceString(parentNode.from, parentNode.to);
             const isN = isNumber(identifierText);
 
-            const attributeOptions = Object.values(Attributes).map(s => ({label: s, apply: `${s} `}));
+            const attributeOptions = Object.values(Attributes)
+              .map(s => `attr:${s}`)
+              .map(s => ({label: s, apply: `${s} `}));
```

Comment:
Do we need the double map? 🤔 
```ts
.map(s => ({label: `attr:${s}`, apply: `attr:${s} `}));
```

Reply:
I dont think we need it. I just think it is more readable and dryer.
I can change if you want.

Reply:
Just committed the fix @jorgeepc